relays/bin-substrate/src/finality_pipeline.rs

```diff
@@ -46,6 +46,9 @@ pub trait SubstrateFinalitySyncPipeline: FinalitySyncPipeline {
 		Ok(params)
 	}
 
+	/// Start finality relay guards.
+	fn start_relay_guards(_target_client: &Client<Self::TargetChain>) {}
```

Comment:
Is the empty default impl intentional? If so, can you add a comment explaining why?